src/clientAuth/bearerBody.js

```diff
@@ -8,12 +14,11 @@ class BearerBody extends ClientAuthnMethod {
   }
 
   /**
-   * More complicated logic then I would have liked it to be
-   *@param {*} cis Request class instance
-   *@param {*} ci Client information
-   *@param {*} requestArgs Request arguments
-   *@param {*} httpArgs HTTP header arguments
-   *@param {*} kwargs
+   *@param {ResourceRequest} cis Request class instance
```

Comment:
This is implicitly nullable and not recommended. Use this:

`@param {!ResourceRequest}` to make it non nullable,
`@param {?ResourceRequest}` to make it nullable.

Same goes for the other objects below. Only primitives (`string`, `boolean`) are non null by default.

Reply:
Fixed